translations/locales/en-US/translations.json

```diff
@@ -365,7 +365,7 @@
     "Or": "Or",
     "AlreadyHave": "Already have an account?",
     "Login": "Log In",
-    "Warning": "By signing up, you agree to the p5.js Editor's <0>Terms of Use</0> and <1>Privacy Policy.</1>"
+    "Warning": " you agree to the p5.js Editor's <0>Terms of Use</0> and <1>Privacy Policy.</1>"
```

Comment:
"By signing up," should be added back in here!

Reply:
oho yeah my bad ,i missed it, should i add it ?

Reply:
Yes, ideally there should be no changes to this file!

Reply:
Hello raclim i fixed the above error .
![image](https://github.com/processing/p5.js-web-editor/assets/116539402/c72fc786-a6d3-4f7c-81ed-ab3283e3006a)
![image](https://github.com/processing/p5.js-web-editor/assets/116539402/08cdc33c-6e20-4a56-bc5c-556a77bb3cbb)

Reply:
Thanks, looks good! I also added in a minor change that updates "Signing" to be lowercase instead of capitalized.